### PR TITLE
Update migrate-ad-fs-application-overview.md

### PR DESCRIPTION
**Pull request by jsoveral**
Part of ICM 456373796 escalation. Update to clarify when the app migration insights get update after customer makes an update.

### CODE REVIEW

**Comment by Court72** [comment]
@omondiatieno

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by omondiatieno — APPROVED** [review]
LGTM!

**Comment by omondiatieno** [comment]
#sign-off

File: docs/identity/enterprise-apps/migrate-ad-fs-application-overview.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ Applications are categorized into following migration statuses:
 
 Application readiness is evaluated based on following predefined AD FS application configuration tests. The tests are run automatically and the results are displayed in the AD FS application migration dashboard as a **Migration status**. If the AD FS configuration isn't compatible with a Microsoft Entra configuration, you get specific guidance on how to address the configuration in Microsoft Entra ID.
 
+## AD FS application migration insights status updates
+
+Once there is an update to the application, internal agents syncs the updates within few minutes. However, AD FS migration insights jobs responsible for evaluating the updates and computes new migration status. Those jobs are scheduled to run every 24 hours, which means data will be computed only once in a day at around 00:00 UTC timezone.
```
Comment (ShannonLeavitt) [inline docs/identity/enterprise-apps/migrate-ad-fs-application-overview.md:54]:
```suggestion
When the application is updated, internal agents sync the updates within few minutes. However, AD FS migration insights jobs are responsible for evaluating the updates and compute a new migration status. Those jobs are scheduled to run every 24 hours, which means that the data will be computed only once in a day, at around 00:00 Coordinated Universal Time (UTC).
```
Edits for Microsoft style and grammar

File: docs/identity/enterprise-apps/migrate-ad-fs-application-overview.md [hunk 2]
```diff
@@ -49,6 +49,10 @@ Applications are categorized into following migration statuses:
 
 Application readiness is evaluated based on following predefined AD FS application configuration tests. The tests are run automatically and the results are displayed in the AD FS application migration dashboard as a **Migration status**. If the AD FS configuration isn't compatible with a Microsoft Entra configuration, you get specific guidance on how to address the configuration in Microsoft Entra ID.
 
+## AD FS application migration insights status updates
+
+When the application is updated, internal agents sync the updates within few minutes. However, AD FS migration insights jobs are responsible for evaluating the updates and compute a new migration status. Those jobs are scheduled to run every 24 hours, which means that the data will be computed only once in a day, at around 00:00 Coordinated Universal Time (UTC).
```
Comment (ShannonLeavitt) [inline docs/identity/enterprise-apps/migrate-ad-fs-application-overview.md:54]:
```suggestion
When the application is updated, internal agents sync the updates within a few minutes. However, AD FS migration insights jobs are responsible for evaluating the updates and compute a new migration status. Those jobs are scheduled to run every 24 hours, which means that the data will be computed only once in a day, at around 00:00 Coordinated Universal Time (UTC).
```

**Pull request merged**